hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2024 Steven Gillies.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
My only comment is about license (verb) vs licence (noun) in British English. Not sure if this requires updates to change it from American English

Reply:
its the standard apache licence, i'm just going to leave it in this instance.